rasa/core/brokers/kafka.py

```diff
@@ -7,10 +7,15 @@
 from rasa.core.brokers.broker import EventBroker
 from rasa.shared.utils.io import DEFAULT_ENCODING
 from rasa.utils.endpoints import EndpointConfig
+from rasa.shared.exceptions import RasaException
 
 logger = logging.getLogger(__name__)
 
 
+class KafkaProducerInitializationError(RasaException):
+    """Raised if the Kafka Producer cannot be properly initialized."""
+
+
 class KafkaEventBroker(EventBroker):
```

Comment:
@wochinge Unrelated to this PR, but a CI check is failing because of the docs-string thing. Should all the arg comments after __init__ be moved up to line 20?

Reply:
No, we usually just write a short docstring for the class itself, and the args should still be where they are right now.